### ISSUE TITLE
Failed to init feature SnapchatPlus

### ISSUE THREAD

**Issue by LolekLiam** [rhunk/SnapEnhance#606]
### Description

Everytime I open Snapchat, it says "Failed to init feature SnapchatPlus". 
The thing is, SnapchatPlus didn't work the first time I installed SnapEnhance, but it worked after I re-installed Snapchat. Now, the issue appeared again. 

What i have tried:
-Clearing the mappings file
-Clearing the cache(both SnapEnhance and Snapchat)
-Force stopping Snapchat

### Reproduction steps

1. Open Snapchat
2. Wait to generate mappings
3. See error

### Logs

```markdown
----part 6 start----
[ 2024-01-23T13:18:33.894    10366: 12327: 12327 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:18:34.210    10366: 12327: 12327 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:18:34.560    10612: 12415: 12415 I/LSPosed-Bridge  ] Loading legacy module me.rhunk.snapenhance from /data/app/~~5gwQN83tNOCs39CK-UXxQg==/me.rhunk.snapenhance-BhbYL6tYzow2dt7gT7V1Hw==/base.apk
[ 2024-01-23T13:18:34.992    10612: 12415: 12415 I/LSPosed-Bridge  ]   Loading class me.rhunk.snapenhance.XposedLoader
[ 2024-01-23T13:18:38.862     5009: 12931: 12931 I/LSPosed-Bridge  ] Loading legacy module io.mesalabs.knoxpatch from /data/app/~~yh3RLt1KcQXtdvuTN5fe6g==/io.mesalabs.knoxpatch-YuDmlLLqVLlEXIj-Ea_-EA==/base.apk
[ 2024-01-23T13:18:38.881     5009: 12931: 12931 I/LSPosed-Bridge  ]   Loading class io.mesalabs.knoxpatch.MainHook_YukiHookXposedInit
[ 2024-01-23T13:18:39.329     5009: 12931: 12931 I/LSPosed-Bridge  ] [KnoxPatch][D][com.samsung.android.scloud] SamsungKeystoreHooks: onHook: loaded.
[ 2024-01-23T13:18:40.161    10279: 13105: 13105 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:18:40.244    10279: 13105: 13105 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:18:41.432    10619: 13264: 13264 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:18:41.499    10619: 13264: 13264 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:18:41.510    10619: 13264: 13264 I/LSPosed-Bridge  ] Loading legacy module me.rhunk.snapenhance from /data/app/~~5gwQN83tNOCs39CK-UXxQg==/me.rhunk.snapenhance-BhbYL6tYzow2dt7gT7V1Hw==/base.apk
[ 2024-01-23T13:18:41.640    10619: 13264: 13264 I/LSPosed-Bridge  ]   Loading class me.rhunk.snapenhance.XposedLoader
[ 2024-01-23T13:18:48.354    10281: 14076: 14076 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:18:48.848    10281: 14076: 14076 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:10.629    10619: 13264: 13484 I/LSPosed-Bridge  ] Generated mappings in 20801 ms
[ 2024-01-23T13:19:11.632    10619: 14727: 14727 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:11.710    10619: 14727: 14727 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:11.725    10619: 14727: 14727 I/LSPosed-Bridge  ] Loading legacy module me.rhunk.snapenhance from /data/app/~~5gwQN83tNOCs39CK-UXxQg==/me.rhunk.snapenhance-BhbYL6tYzow2dt7gT7V1Hw==/base.apk
[ 2024-01-23T13:19:11.877    10619: 14727: 14727 I/LSPosed-Bridge  ]   Loading class me.rhunk.snapenhance.XposedLoader
[ 2024-01-23T13:19:12.812    10619: 14727: 14727 I/LSPosed-Bridge  ] Failed to init feature SnapchatPlus
[ 2024-01-23T13:19:12.812    10619: 14727: 14727 E/LSPosed-Bridge  ] java.lang.Exception: No mapping found for SubscriptionInfoClass
	at me.rhunk.snapenhance.manager.impl.MappingManager.getMappedObject(MappingManager.kt:155)
	at me.rhunk.snapenhance.manager.impl.MappingManager.getMappedClass(MappingManager.kt:163)
	at me.rhunk.snapenhance.features.impl.tweaks.SnapchatPlus.init(SnapchatPlus.kt:16)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$initializeFeatures$1.invoke(FeatureManager.kt:122)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$initializeFeatures$1.invoke(FeatureManager.kt:122)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$featureInitializer$2$callback$1.invoke-d1pmJ48(FeatureManager.kt:104)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$featureInitializer$2$callback$1.invoke(FeatureManager.kt:102)
	at me.rhunk.snapenhance.manager.impl.FeatureManager.featureInitializer(FeatureManager.kt:111)
	at me.rhunk.snapenhance.manager.impl.FeatureManager.initializeFeatures(FeatureManager.kt:122)
	at me.rhunk.snapenhance.manager.impl.FeatureManager.init(FeatureManager.kt:97)
	at me.rhunk.snapenhance.SnapEnhance.init(SnapEnhance.kt:103)
	at me.rhunk.snapenhance.SnapEnhance.access$init(SnapEnhance.kt:18)
	at me.rhunk.snapenhance.SnapEnhance$init$1.invokeSuspend(Unknown Source:14)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
	at me.rhunk.snapenhance.SnapEnhance$1$4$1.invoke(SnapEnhance.kt:51)
	at me.rhunk.snapenhance.SnapEnhance$1$4$1.invoke(SnapEnhance.kt:44)
	at me.rhunk.snapenhance.bridge.BridgeClient.start(BridgeClient.kt:62)
	at me.rhunk.snapenhance.SnapEnhance$1.invoke(SnapEnhance.kt:44)
	at me.rhunk.snapenhance.SnapEnhance$1.invoke(SnapEnhance.kt:29)
	at me.rhunk.snapenhance.hook.Hooker$hook$$inlined$hook$1.beforeHookedMethod(Hooker.kt:16)
	at hpNCpnQkMUl.jgid.mKFzG.XposedBridge$LegacyApiSupport.handleBefore(Unknown Source:24)
	at J.callback(Unknown Source:179)
	at LSPHooker_.attach(Unknown Source:11)
	at android.app.Instrumentation.newApplication(Instrumentation.java:1178)
	at android.app.LoadedApk.makeApplication(LoadedApk.java:1456)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7458)
	at android.app.ActivityThread.access$1600(ActivityThread.java:310)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2281)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8663)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
[ 2024-01-23T13:19:13.132    10323: 14869: 14869 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:13.281    10323: 14869: 14869 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:16.555    10413: 15157: 15157 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:16.617    10413: 15157: 15157 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:20.292    10377: 15389: 15389 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:20.371    10377: 15389: 15389 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:21.175    10300: 15427: 15427 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:21.254    10300: 15427: 15427 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:22.309    10320: 15594: 15594 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:22.476    10320: 15594: 15594 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:24.411    10358: 15914: 15914 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:24.470    10279: 15924: 15924 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:24.515    10358: 15914: 15914 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:24.555    10279: 15924: 15924 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:25.198    10309: 15994: 15994 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:25.270    10309: 15994: 15994 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:27.369    10619: 16198: 16198 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:27.434    10619: 16198: 16198 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:27.447    10619: 16198: 16198 I/LSPosed-Bridge  ] Loading legacy module me.rhunk.snapenhance from /data/app/~~5gwQN83tNOCs39CK-UXxQg==/me.rhunk.snapenhance-BhbYL6tYzow2dt7gT7V1Hw==/base.apk
[ 2024-01-23T13:19:27.625    10619: 16198: 16198 I/LSPosed-Bridge  ]   Loading class me.rhunk.snapenhance.XposedLoader
[ 2024-01-23T13:19:29.305    10612: 16339: 16339 I/LSPosed-Bridge  ] Loading legacy module me.rhunk.snapenhance from /data/app/~~5gwQN83tNOCs39CK-UXxQg==/me.rhunk.snapenhance-BhbYL6tYzow2dt7gT7V1Hw==/base.apk
[ 2024-01-23T13:19:29.484    10612: 16339: 16339 I/LSPosed-Bridge  ]   Loading class me.rhunk.snapenhance.XposedLoader
[ 2024-01-23T13:19:30.174    10619: 16198: 16198 I/LSPosed-Bridge  ] Failed to init feature SnapchatPlus
[ 2024-01-23T13:19:30.175    10619: 16198: 16198 E/LSPosed-Bridge  ] java.lang.Exception: No mapping found for SubscriptionInfoClass
	at me.rhunk.snapenhance.manager.impl.MappingManager.getMappedObject(MappingManager.kt:155)
	at me.rhunk.snapenhance.manager.impl.MappingManager.getMappedClass(MappingManager.kt:163)
	at me.rhunk.snapenhance.features.impl.tweaks.SnapchatPlus.init(SnapchatPlus.kt:16)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$initializeFeatures$1.invoke(FeatureManager.kt:122)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$initializeFeatures$1.invoke(FeatureManager.kt:122)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$featureInitializer$2$callback$1.invoke-d1pmJ48(FeatureManager.kt:104)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$featureInitializer$2$callback$1.invoke(FeatureManager.kt:102)
	at me.rhunk.snapenhance.manager.impl.FeatureManager.featureInitializer(FeatureManager.kt:111)
	at me.rhunk.snapenhance.manager.impl.FeatureManager.initializeFeatures(FeatureManager.kt:122)
	at me.rhunk.snapenhance.manager.impl.FeatureManager.init(FeatureManager.kt:97)
	at me.rhunk.snapenhance.SnapEnhance.init(SnapEnhance.kt:103)
	at me.rhunk.snapenhance.SnapEnhance.access$init(SnapEnhance.kt:18)
	at me.rhunk.snapenhance.SnapEnhance$init$1.invokeSuspend(Unknown Source:14)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
	at me.rhunk.snapenhance.SnapEnhance$1$4$1.invoke(SnapEnhance.kt:51)
	at me.rhunk.snapenhance.SnapEnhance$1$4$1.invoke(SnapEnhance.kt:44)
	at me.rhunk.snapenhance.bridge.BridgeClient.start(BridgeClient.kt:62)
	at me.rhunk.snapenhance.SnapEnhance$1.invoke(SnapEnhance.kt:44)
	at me.rhunk.snapenhance.SnapEnhance$1.invoke(SnapEnhance.kt:29)
	at me.rhunk.snapenhance.hook.Hooker$hook$$inlined$hook$1.beforeHookedMethod(Hooker.kt:16)
	at hpNCpnQkMUl.jgid.mKFzG.XposedBridge$LegacyApiSupport.handleBefore(Unknown Source:24)
	at J.callback(Unknown Source:179)
	at LSPHooker_.attach(Unknown Source:11)
	at android.app.Instrumentation.newApplication(Instrumentation.java:1178)
	at android.app.LoadedApk.makeApplication(LoadedApk.java:1456)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7458)
	at android.app.ActivityThread.access$1600(ActivityThread.java:310)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2281)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8663)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
[ 2024-01-23T13:19:36.359    10619: 17250: 17250 I/LSPosed-Bridge  ] Loading legacy module ru.zsonmomb.kluzoleub from /data/app/~~t1VXu7g_0Rjh68BuTo-qJg==/ru.zsonmomb.kluzoleub-JJtX7DHhYBJpz6Ts7sPPWA==/base.apk
[ 2024-01-23T13:19:36.457    10619: 17250: 17250 I/LSPosed-Bridge  ]   Loading class com.xposed.XSupport
[ 2024-01-23T13:19:36.474    10619: 17250: 17250 I/LSPosed-Bridge  ] Loading legacy module me.rhunk.snapenhance from /data/app/~~5gwQN83tNOCs39CK-UXxQg==/me.rhunk.snapenhance-BhbYL6tYzow2dt7gT7V1Hw==/base.apk
[ 2024-01-23T13:19:36.670    10619: 17250: 17250 I/LSPosed-Bridge  ]   Loading class me.rhunk.snapenhance.XposedLoader
[ 2024-01-23T13:19:38.348    10612: 17459: 17459 I/LSPosed-Bridge  ] Loading legacy module me.rhunk.snapenhance from /data/app/~~5gwQN83tNOCs39CK-UXxQg==/me.rhunk.snapenhance-BhbYL6tYzow2dt7gT7V1Hw==/base.apk
[ 2024-01-23T13:19:38.506    10612: 17459: 17459 I/LSPosed-Bridge  ]   Loading class me.rhunk.snapenhance.XposedLoader
[ 2024-01-23T13:19:39.036    10619: 17250: 17250 I/LSPosed-Bridge  ] Failed to init feature SnapchatPlus
[ 2024-01-23T13:19:39.037    10619: 17250: 17250 E/LSPosed-Bridge  ] java.lang.Exception: No mapping found for SubscriptionInfoClass
	at me.rhunk.snapenhance.manager.impl.MappingManager.getMappedObject(MappingManager.kt:155)
	at me.rhunk.snapenhance.manager.impl.MappingManager.getMappedClass(MappingManager.kt:163)
	at me.rhunk.snapenhance.features.impl.tweaks.SnapchatPlus.init(SnapchatPlus.kt:16)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$initializeFeatures$1.invoke(FeatureManager.kt:122)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$initializeFeatures$1.invoke(FeatureManager.kt:122)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$featureInitializer$2$callback$1.invoke-d1pmJ48(FeatureManager.kt:104)
	at me.rhunk.snapenhance.manager.impl.FeatureManager$featureInitializer$2$callback$1.invoke(FeatureManager.kt:102)
	at me.rhunk.snapenhance.manager.impl.FeatureManager.featureInitializer(FeatureManager.kt:111)
	at me.rhunk.snapenhance.manager.impl.FeatureManager.initializeFeatures(FeatureManager.kt:122)
	at me.rhunk.snapenhance.manager.impl.FeatureManager.init(FeatureManager.kt:97)
	at me.rhunk.snapenhance.SnapEnhance.init(SnapEnhance.kt:103)
	at me.rhunk.snapenhance.SnapEnhance.access$init(SnapEnhance.kt:18)
	at me.rhunk.snapenhance.SnapEnhance$init$1.invokeSuspend(Unknown Source:14)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
	at me.rhunk.snapenhance.SnapEnhance$1$4$1.invoke(SnapEnhance.kt:51)
	at me.rhunk.snapenhance.SnapEnhance$1$4$1.invoke(SnapEnhance.kt:44)
	at me.rhunk.snapenhance.bridge.BridgeClient.start(BridgeClient.kt:62)
	at me.rhunk.snapenhance.SnapEnhance$1.invoke(SnapEnhance.kt:44)
	at me.rhunk.snapenhance.SnapEnhance$1.invoke(SnapEnhance.kt:29)
	at me.rhunk.snapenhance.hook.Hooker$hook$$inlined$hook$1.beforeHookedMethod(Hooker.kt:16)
	at hpNCpnQkMUl.jgid.mKFzG.XposedBridge$LegacyApiSupport.handleBefore(Unknown Source:24)
	at J.callback(Unknown Source:179)
	at LSPHooker_.attach(Unknown Source:11)
	at android.app.Instrumentation.newApplication(Instrumentation.java:1178)
	at android.app.LoadedApk.makeApplication(LoadedApk.java:1456)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7458)
	at android.app.ActivityThread.access$1600(ActivityThread.java:310)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2281)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8663)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
```

### Snapchat Version

12.70.0.34

### SnapEnhance Version

1.2.5

### Agreement

- [X] This is not a bug regarding Snapchat+.
- [X] I have provided a detailed description of the issue.
- [X] I have attached a log if deemed neccessary.
- [X] This issue is not a duplicate.

**Comment by n00b69** [comment]
I came here to report the same issue, only to find out someone already beat me to it

**Comment by rhunk** [comment]
versions 1.x.x are no longer updated, and the bug has already been fixed in [v2.0.0](https://t.me/snapenhance_ci)

**Comment by Jnss98b** [comment]
> versions 1.x.x are no longer updated, and the bug has already been fixed in [v2.0.0](https://t.me/snapenhance_ci)

Does this mean there won't be new releases on github? @rhunk
